packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -28,6 +28,105 @@ Email templates use common and unique [Expression Language (EL) variables](https
+
+##### EA Feature requirements
+
+To access this EA feature, you need the following prerequisites:
+
+* Requires an Okta preview org (`example.oktapreview.com`) and Okta production org (`example.okta.com`)
+* Requires the self-service feature flag, `ENHANCED_EMAIL_MACROS`, to be enabled for the org
+
+> **Note:** This feature is supported in Okta Identity Engine. <ApiLifecycle access="ie" />
+
+#### Velocity Templating syntax
+
+The templating syntax that is used in enhanced email customization is the Velocity Templating Language (VTL). Variables that are to be interpolated in the content of the template are preceded by a dollar sign. Dot notation is used to reference sub-objects.
+
+The new templating syntax is different from the EL expression-based Okta email templating syntax that was previously used. You no longer need to use curly braces around the variable name.
+
+All template variables that were previously available in the email templating feature are still available in the new feature, but you need to use the new syntax to reference variables.
+
+##### Variable syntax change example
+
+Previously with EL syntax, you could reference the first name of the user by using `${user.firstName}`. Now with the Velocity Templating syntax, you would use `$user.profile.firstName`.
+
+The previously available template variables are listed in [Customization Variables](https://help.okta.com/en/prod/Content/Topics/Settings/Settings_Email.htm).
+
+##### Function syntax change example
+
+This EA feature supports the use of functions within email templates, but how you call the function's syntax is different. For example, instead of calling functions in the EL templating syntax like this:
+
+`${f:formatTimeDiffHoursNowInUserLocale(org.activationTokenExpirationHours)}`
+
+You would call functions in the Velocity Templating syntax like this:
+
+`$f.formatTimeDiffHoursNowInUserLocale($org.activationTokenExpirationHours)`
+
+#### Access to all User Profile attributes
+
+You can now reference any attribute in a user's Okta User Profile in your email templates. The notation is `$user.profile.attributeName`. For example, to reference the displayName attribute of the user's Okta user profile, you would use `$user.profile.displayName`.
+
+See [Profile object](/docs/api/resources/users#profile-object) for more information on the available user profile attributes.
+
+##### Example of user attributes
+
+###### Registration: Email verification with customer's preferred language
+Trigger an email in your end-users' preferred language by using the conditional logic and calling the `preferredLanguage` attribute.
+
+###### Activation department specific email
+If a customer wants to specify a department such as Engineering in an Activation Email, they are able to call the user attribute `department`.
+
+#### Access to org attributes
+
+You can also reference some org-level attributes:
```

Comment:
Maybe add "such as:"

Reply:
Can we include a link a comprehensive list of variables and properties that can be used in templates?

Reply:
@gormanho-okta I asked Christine to punt on this comment, I think its a great suggestion for a next iteration just so we can get this released ASAP